.github/workflows/kubevirtci-tag-bumper.yaml

```diff
@@ -0,0 +1,53 @@
+name: Bump KubevirtCI Tag
+
+on:
+  schedule:
+    - cron: '0 0 1 1,5,9 *'  # 00:00 on the 1st of Jan, May, Sep
+  workflow_dispatch:
+
+jobs:
+  bump-kubevirtci:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Git config
+        run: |
+          git config user.name "github-actions[bot]"
+          git config user.email "github-actions[bot]@users.noreply.github.com"
+
+      - name: Run bump-kubevirtci.sh
+        run: |
+          ./hack/bump-kubevirtci.sh
+
+      - name: Check for changes
+        id: changes
+        run: |
+          if git diff --quiet cluster/cluster.sh; then
+            echo "changed=false" >> $GITHUB_OUTPUT
+          else
+            echo "changed=true" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Extract new tag from cluster.sh
+        if: steps.changes.outputs.changed == 'true'
+        id: extract
+        run: |
+          NEW_TAG=$(grep -oP 'KUBEVIRTCI_TAG:-\K[^}]+' cluster/cluster.sh)
+          echo "Detected new tag: $NEW_TAG"
+          echo "tag=$NEW_TAG" >> $GITHUB_OUTPUT
+
+      - name: Create Pull Request
+        if: steps.changes.outputs.changed == 'true'
```

Comment:
can we compact those steps to only one ?
maybe directly `if git diff ...; do stuff..` ?

Reply:
no, because this step (by peter-evans) is a standalone gitActions building block. If we wanted to bulk it together, we'd need to implement it ourselves like in the auto bumper case. But that's not necessary.